CONTRIBUTING.md

```diff
@@ -83,5 +83,5 @@ Before we can accept your pull request, you need to [sign our CLA](https://grafa
 
 - Set up your [development environment](contribute/developer-guide.md).
 - Learn how to [contribute to our documentation](contribute/documentation/README.md).
-- Get started [developing plugins](/developers/plugin-tools) for Grafana.
+- Get started [developing plugins](https://grafana.com/developers/plugin-tools) for Grafana.
```

Comment:
```suggestion
- Get started [developing plugins](/developers/plugin-tools) for Grafana.
```

Reply:
This is the one we definitely want to keep.